src/data/types.ts

```diff
@@ -80,14 +80,12 @@ export type Project = {
     website?: string;
 
      /**
-     * Link to deployed app on a store
+     * If the project is an app link to the app and type of app. 
      */
-    appDownloadLink?: string;
-
-    /**
-     * Type of app
-     */
-    appType?: AppType;
+    app?: {
+      url?: string;
+      appType: AppType;
```

Comment:
Let's just inline `AppType` here and get rid of the `AppType` (streamline the docs a bit haha)

```suggestion
      type: 'ios' | 'android';
```

Reply:
Done!

Reply:
Note that in general, try to avoid repeating namespaces, for example:

`app.type` is already the app's type, but `app.appType` just adds redundant namespace `app` again when you can already see it is the app's type

Reply:
I see.. sorry missed the comment